File: 75-Blind/TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] solutions = new int[2];
        HashMap<Integer, Integer> myMap = new HashMap<>();
        
        int comp;
        for (int i = 0; i < nums.length; i++) {
            comp = target - nums[i];
            if (myMap.containsKey(comp)) {
                solutions[0] = i;
                solutions[1] = myMap.get(comp);
                break;
            } else {
                myMap.put(nums[i], i);
            }
        }
        return solutions;
    }
}

File: 75-Blind/Python/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        my_dict = {}
        
        for i in range (len(nums)):
            comp = target - nums[i];
            if comp in my_dict:
                return [i, my_dict.get(comp)]
            my_dict[nums[i]] = i;
        
        return [-1, -1]
